modules/lang-painless/src/test/resources/rest-api-spec/test/painless/80_script_score.yml

```diff
@@ -496,3 +496,53 @@
     - match: { hits.total: 2 }
     - match: { hits.hits.0._id : "2" }
     - match: { hits.hits.1._id : "1" }
+
+
+---
+"Script Score With Highlight":
+  - skip:
+      version: " - 7.4.99"
+      reason: "highlight for script_score was introduce in 7.5"
```

Comment:
Small typo: introduce -> introduced. Also, should this be `7.99.99` for now since it won't be available on 7.5 until it's backported?

Reply:
@jtibshirani Thanks for the review. I have changed the version to 7.4.1.  

> should this be 7.99.99 for now since it won't be available on 7.5 until it's backported?
Looks like there are no bwc tests for modules:lang-painless, as the build passed.  Looks like it is safe without backport.  But I am going to backport the change immediately after the merge to master.